### PR TITLE
Add draft Interim Finance Ctte policy docs.

### PR DESCRIPTION
**Pull request by jdswinbank**
Addressing #56.

### CODE REVIEW

File: finance/finance-ctte-policy.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+Role and responsibilities of the Committee
+==========================================
+
+#. The Committee is not responsible for deciding on what the highest priority or most appropriate uses for Project funds are.
+   In this respect, it must defer to to the Coordination Committee.
+
+   (Agreed `2020-02-07`__.)
+
+__ https://docs.google.com/document/d/193ERJ6wMNPaNjGSWbS94KHGnojpZ_zwXxWTi9L2etSo/edit#bookmark=id.fwmv8h9ysndt
```
Comment (eteq) [inline finance/finance-ctte-policy.rst:17]:
```suggestion
   __ https://docs.google.com/document/d/193ERJ6wMNPaNjGSWbS94KHGnojpZ_zwXxWTi9L2etSo/edit#bookmark=id.fwmv8h9ysndt
```

File: finance/finance-ctte-policy.rst [hunk 2]
```diff
@@ -0,0 +1,58 @@
+##################################
+Interim Finance Committee Policies
+##################################
+
+This document records policies adopted by the `Astropy Interim Finance Committee`_ (hereafter “the Committee”).
```
Comment (eteq) [inline finance/finance-ctte-policy.rst:5]: Maybe add a bit of context that this is a "running policy" document as opposed to something that's very fixed.

File: finance/finance-ctte-policy.rst [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+   __ https://docs.google.com/document/d/193ERJ6wMNPaNjGSWbS94KHGnojpZ_zwXxWTi9L2etSo/edit#bookmark=id.fwmv8h9ysndt
+
+#. The Committee should notify the Coordination Committee where they believe the priorities set by the latter may not be aligned with the views of the wider community.
+   This is particularly important when members of the Committee may, as part of Committee business, have access to information which is not available to the community at large.
+
+   (Agreed `2020-02-07`__.)
+
+   __ https://docs.google.com/document/d/193ERJ6wMNPaNjGSWbS94KHGnojpZ_zwXxWTi9L2etSo/edit#bookmark=id.fwmv8h9ysndt
+
+#. The Committee will consider funding requests submitted according to the policy documented at https://github.com/astropy/project/blob/master/finance/funding_requests.md.
```
Comment (kelle) [inline finance/finance-ctte-policy.rst:33]: The Committee will allocated funding as directed by the CoCo and will consider requests submitted ...

**Comment by kelle** [comment]
I added a comment to #4 based on our conversations this week. It would be great to work on this and get this merged this week.

**Pull request merged**